nabs.yml

```diff
@@ -1,3 +1,9 @@
 test:
-  lint: eslint .
+  lint: eslint src
```

Comment:
This line needs to be reverted.

It might be easier if you actually use nabs to generate the package.json's script tags :)

Reply:
This line actually still needs to be reverted.

Reply:
Yes, but the nabs.yml (which generates the scripts in package.json) says:
```yaml
test:
  lint: eslint src
```